nuxeo-web-ui-ftest/test/pages/ui/oauth2/user_authorized_apps.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+import BasePage from '../../base';
+
+class App {
```

Comment:
I don't like the name, put something more specific if possible

Reply:
I can call it AuthorizedApp, but keep it mind that this class won't be exported.